lab 1/src/main/java/com/example/myapp/MainActivity.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.myapp
+//import android.R
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+
+
+class MainActivity : AppCompatActivity() {
+
+    companion object{
+        const val USER = 0
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+    }
+
+    fun change(view: View){
+        val myToast = Toast.makeText(this, "changed!", Toast.LENGTH_SHORT)
+        val textView: TextView = findViewById(R.id.textView) as TextView
+        val string = textView.text.toString().substringBefore(" ") + " Marry!"
+//        println("string = " + string)
+        textView.setText(string)
+        myToast.show()
+    }
+
+    fun send(view: View){
+//        val myToast = Toast.makeText(this, "sent!", Toast.LENGTH_SHORT)
+//        myToast.show();

```

Comment:
@KateCotelnic remove all commented code. It is dead code which does not add any value to your project